### PR TITLE
RT-321: Fix missing slugs in PROD

### PR DESCRIPTION
**Pull request by dev-digital**
**Descriptions**
Data ingested using _ingest_csv_ admin command found to be missing the models specific over-rides. Though data has been successfully ingested and saved to db, app wasn't not able to consume this data which has been addressed in this PR.

**Changes**
- Call _save_ on each of the ingested model to trigger custom over-rides within _ingest_csv_
- StrategicAction.supporitng_organisations changed to ArrayField to accommodate real data
- SupplyChain.vulnerability moved to TextChoices of type RAGRating
- Related changes within e2e and unit tests

**Note**
- For convenience I have deployed this branch to _dev_ environment along with PROD data for us to test/investigate. I have taken backup of previous _dev_ data and revert to that when appropriate.
- There could be some follow up tasks based on this work(like revisiting Download csv) to see if any further changes are required.

### CODE REVIEW

File: update_supply_chain_information/supply_chains/test/data/supply_chain_sample.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-id,name,contact_name,contact_email,vulnerability_status,vulnerability_status_disagree_reason,risk_severity_status,risk_severity_status_disagree_reason,is_archived,archived_date,last_submission_date,gov_department
-fecad835-63f8-4a60-b408-b80c10f80bef,medicines,,,red,,,high,0,,4/30/21,07622ccf-ecc5-49bb-949a-0064ab996fa1
-4c3b8ef7-f36c-4280-9f59-ee74025b6751,medical devices and consumables,,,amber,,,high,0,,4/30/21,07622ccf-ecc5-49bb-949a-0064ab996fa1
-59a60a43-ac85-4113-9a39-35d6709a3a31,vaccines,,,red,,,high,0,,4/30/21,07622ccf-ecc5-49bb-949a-0064ab996fa1
\ No newline at end of file
+id,name,contact_name,contact_email,vulnerability_status,vulnerability_status_disagree_reason,risk_severity_status_disagree_reason,risk_severity_status,is_archived,archived_date,last_submission_date,gov_department
```
Comment (dev-digital) [inline update_supply_chain_information/supply_chains/test/data/supply_chain_sample.csv:1]: Update from latest csvs from gitlab repo

File: update_supply_chain_information/supply_chains/templates/strategic_action_summary.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h2 class="govuk-accordion__section-heading">
               Which other government departments are supporting this strategic action?
             </dt>
             <dd class="govuk-summary-list__value">
-              {{ action.get_supporting_organisations_display|default:"No information" }}
+              {{ action.supporting_organisations|join:", "|default:"No information" }}
```
[the remark this answers is not in the document]
Comment (NickFitz) [reply update_supply_chain_information/supply_chains/templates/strategic_action_summary.html:79]: From a quick search, it looks as if ArrayField had had some known issues with implementing `get_FOO_display`. This seems like a reasonable alternative for our case :)

**Review by NickFitz — APPROVED** [review]
Seems OK to me 👍

File: update_supply_chain_information/supply_chains/test/factories.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ class StrategicActionFactory(factory.django.DjangoModelFactory):
     impact = factory.Faker("text")
     category = factory.fuzzy.FuzzyChoice(StrategicAction.Category)
     geographic_scope = factory.fuzzy.FuzzyChoice(StrategicAction.GeographicScope)
-    supporting_organisations = factory.fuzzy.FuzzyChoice(StrategicAction.SupportingOrgs)
+    supporting_organisations = ["DIT"]
```
[the remark this answers is not in the document]
Comment (mforner13) [reply update_supply_chain_information/supply_chains/test/factories.py:34]: Just tried for a while and couldn't figure it out either :p I think this option is fine though!

**Review by mforner13 — APPROVED** [review]
👍

**Pull request merged**